program_info/org.polymc.PolyMC.svg

```diff
@@ -1,9 +1,21 @@
-<path stroke="#000000" d="M19 16h2M18 17h4M17 18h6M16 19h9M17 20h9M19 21h9M20 22h9M44 22h6M21 23h4M27 23h3M42 23h8M22 24h3M27 24h3M40 24h12M15 25h3M22 25h9M39 25h13M12 26h1M15 26h2M22 26h9M38 26h2M48 26h6M12 27h4M21 27h11M38 27h2M48 27h6M10 28h2M19 28h13M37 28h1M50 28h6M10 29h2M17 29h17M37 29h1M50 29h6M9 30h4M17 30h7M26 30h8M51 30h6M9 31h3M18 31h4M26 31h10M52 31h5M9 32h3M28 32h8M52 32h5M9 33h3M28 33h10M52 33h5M9 34h5M30 34h8M50 34h7M9 35h5M30 35h10M50 35h6M10 36h6M24 36h2M32 36h8M48 36h8M10 37h7M23 37h2M32 37h10M47 37h9M10 38h15M34 38h20M10 39h14M34 39h20M12 40h11M36 40h16M12 41h10M38 41h14M16 42h5M40 42h8M16 43h2M40 43h8" />
-<path stroke="#595959" d="M38 22h6M38 23h4M14 24h8M36 24h4M14 25h1M18 25h4M36 25h3M13 26h2M34 26h4M34 27h4M12 28h2M32 28h5M12 29h2M34 29h3M34 30h3M26 34h4M26 35h4M26 36h6M25 37h7M25 38h5M24 39h6M23 40h3M22 41h4M21 42h3M18 43h6" />
-<path stroke="#488737" d="M25 23h2M25 24h2" />
-</svg>
\ No newline at end of file
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Created with Inkscape (http://www.inkscape.org/) -->
+<svg width="64" height="64" version="1.1" viewBox="0 0 16.933 16.933" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+ <defs>
+  <linearGradient id="linearGradient84726" x1="4.4979" x2="12.435" y1="3.8011" y2="9.5681" gradientUnits="userSpaceOnUse">
+   <stop stop-color="#88b858" offset="0"/>
+   <stop stop-color="#72b147" offset=".5"/>
+   <stop stop-color="#5a9a30" offset="1"/>
+  </linearGradient>
+ </defs>
+ <g>
+  <path d="m3.561 16.016s0-3.5642 4.9056-3.5642c4.9069 0 4.9056 3.5642 4.9056 3.5642z" fill="#765338"/>
+  <path d="m8.4667 12.452-4.9056 3.5642-3.0319-9.3311z" fill="#b7835a"/>
+  <path d="m8.4667 12.452 7.9375-5.7669-3.0319 9.3311z" fill="#5b422d"/>
+  <path d="m8.8308 12.716-0.36417 0.26458-0.36417-0.26458c0-0.26458 0.36417-0.26458 0.36417-0.26458s0.36417 0 0.36417 0.26458z" fill="#72b147"/>
+  <path d="m8.4667 12.452s-2e-7 -5.7669 7.9375-5.7669l-0.22507 0.69269-0.91853 1.1965-0.91853 0.13819-0.91853 1.1965-0.91853 0.13819-0.91853 1.1965-0.91853 0.13819-0.91853 1.1965-0.91853 0.13819z" fill="#5a9a30"/>
+  <path d="m8.1025 12.716-0.91853-0.13819-0.91853-1.1965-0.91853-0.13819-0.91853-1.1965-0.91853-0.13819-0.91853-1.1965-0.91853-0.13819-0.91853-1.1965-0.22507-0.69269c7.9375 1e-7 7.9375 5.7669 7.9375 5.7669z" fill="#88b858"/>
+  <path d="m0.52917 6.6846 7.9375 5.7669 7.9375-5.7669-7.9375-5.7669z" fill="url(#linearGradient84726)"/>
+ </g>
+ <path d="m0.75424 7.3773-0.22507-0.69269 7.9375 5.7669 7.9375-5.7669-0.22507 0.69269-7.7124 5.6034z" fill-opacity="0"/>
+</svg>
```

Comment:
Additionally, there's an extra polymc-logo file

Reply:
theres polymc-logo (light text) and polymc_logo (dark text)

Reply:
Make it polymc-dark.png and polymc-light.png

Reply:
Fixed They are both named accordingly now